synapse/rest/media/v1/oembed.py

```diff
@@ -119,10 +121,22 @@ def parse_oembed_response(self, url: str, raw_body: bytes) -> OEmbedResult:
             # Ensure the cache age is None or an int.
             cache_age = oembed.get("cache_age")
             if cache_age:
-                cache_age = int(cache_age)
+                cache_age = int(cache_age) * 1000
 
             # The results.
-            open_graph_response = {"og:title": oembed.get("title")}
+            open_graph_response = {
+                "og:url": url,
+            }
```

Comment:
Won't this be the oEmbed URL, rather than the original? We seem to be passing the wrong URL in `_do_preview`

Reply:
Quite possibly.

Reply:
Thanks! I've fixed that and updated the tests!